docs/source/cloudlabtutorial/index.rst

```diff
@@ -0,0 +1,126 @@
+Cloudlab Tutorial 
+=====================================
+
+Getting Started
+-----------------
+
+- This tutorial assumes that you have access to the openNetVM `CloudLab <https://cloudlab.us/>`_ 
+    - To gain access, you must first create an account, choose to join a project and enter "GWCloudLab" to request access
+
+Setting up SSH Keys for CloudLab
+-----------------
+
+CloudLab allows you to upload an SSH key
+
+- Open a terminal and cd to the directory you want to store the keys in
+
+    Generally, on a Windows machine, all SSH-related files are found in following location: /c/Users/PC_USER_NAME/.ssh/
```

Comment:
Add a note for MacOS / linux users about where to find this info -- you can just say `~/.ssh`